### PR TITLE
[Close #238] change https protocol as default for deps rule

### LINKED ISSUE #238: The default for git_for_deps_erlang_mk should be https
This goes hand in hand with inaka/erlang_guidelines#51

### PR DESCRIPTION
**Pull request by Euen**
[Close #238] change https protocol as default for deps rule

### CODE REVIEW

File: test/project_SUITE.erl [hunk 1]
```diff
@@ -93,10 +95,11 @@ verify_git_for_deps_erlang_mk(_Config) ->
     Filename = "Makefile.fail",
     {ok, File} = elvis_test_utils:find_file(SrcDirs, Filename),
 
-    [_, _, _] = elvis_project:git_for_deps_erlang_mk(ElvisConfig, File, #{}),
+    [_, _, _, _, _, _] =
+        elvis_project:git_for_deps_erlang_mk(ElvisConfig, File, #{}),
 
     RuleConfig =  #{ignore => [sync]},
-    [_, _] = elvis_project:git_for_deps_erlang_mk(ElvisConfig,
+    [_, _, _, _, _] = elvis_project:git_for_deps_erlang_mk(ElvisConfig,
```
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (102, 21), (112, 21).

File: test/project_SUITE.erl [hunk 2]
```diff
@@ -110,6 +113,32 @@ verify_git_for_deps_erlang_mk(_Config) ->
...
+    SrcDirs = elvis_config:dirs(ElvisConfig),
+
+    Filename = "Makefile.fail",
+    {ok, File} = elvis_test_utils:find_file(SrcDirs, Filename),
+
+    [_, _, _, _, _, _] =
+        elvis_project:protocol_for_deps_erlang_mk(ElvisConfig, File, #{}),
+
+    RuleConfig =  #{ignore => [sync]},
+    [_, _, _, _, _] = elvis_project:protocol_for_deps_erlang_mk(ElvisConfig,
```
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (128, 21), (138, 21).

File: test/project_SUITE.erl [hunk 3]
```diff
@@ -93,10 +95,11 @@ verify_git_for_deps_erlang_mk(_Config) ->
     Filename = "Makefile.fail",
     {ok, File} = elvis_test_utils:find_file(SrcDirs, Filename),
 
-    [_, _, _] = elvis_project:git_for_deps_erlang_mk(ElvisConfig, File, #{}),
+    [_, _, _, _, _, _] =
```
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (98, 5), (124, 5).

Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (98, 5), (124, 5).

File: test/project_SUITE.erl (hunk 2 above)
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (128, 21), (138, 21).

Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (128, 21), (138, 21).

File: test/project_SUITE.erl (hunk 1 above)
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (102, 21), (112, 21).

File: test/project_SUITE.erl (hunk 3 above)
Comment (gadgetci) [inline test/project_SUITE.erl]: According to **Elvis**:
> The code in the following (LINE, COL) locations has the same structure: (98, 5), (124, 5).

File: test/project_SUITE.erl (hunk 2 above)
Comment (jfacorro) [inline test/project_SUITE.erl]: It's ok @elvisci, don't worry about it.

**Pull request merged**